### PR TITLE
Viewer toolbar

### PR DESCRIPTION
**Pull request by Dagonite**
Implementation of the viewer GUI toolbar.

### CODE REVIEW

**Comment by lauramurgatroyd** [comment]
Hi @Dagonite, thanks for opening the PR! We have a training course/hackathon coming up soon for our CIL software, so we are pretty busy over the next few weeks. This means we probably won't get on to looking at this yet, but we will get back to it after then!

File: Wrappers/Python/ccpi/viewer/ui/SettingsDialog.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
+import os
+
+from eqt.ui import FormDialog, UISliderWidget
+from PySide2 import QtCore, QtWidgets
+from vtkmodules.util import colors
```
Comment (lauramurgatroyd) [inline Wrappers/Python/ccpi/viewer/ui/SettingsDialog.py:5]:
```suggestion
from vtk import vtkNamedColors as colors
```

File: Wrappers/Python/ccpi/viewer/ui/helpers.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+from vtkmodules.util import colors
```
Comment (lauramurgatroyd) [inline Wrappers/Python/ccpi/viewer/ui/helpers.py:1]:
```suggestion
from vtk import vtkNamedColors as colors
```

**Review by lauramurgatroyd — CHANGES_REQUESTED** [review]
Unfortunately we cannot yet use imports from vtkmodules as we still have to support vtk8 for the iDVC app.

Instead we need to use: colors = vtk.vtkNamedColors()

This requires some more changes rather than just the import change so I will look into this another time

**Comment by paskino** [comment]
Relevant changes are updated in #347

**Pull request closed without merging**